lib/src/plugin_controller.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_reactive_ble/src/converter/args_to_protubuf_converter.dart';
+import 'package:flutter_reactive_ble/src/model/uuid.dart';
+
+class PluginController {
+  const PluginController(this.converter, this.bleMethodChannel);
```

Comment:
I'd used named parameters here, because they don't seem to be obvious or easy-to-guess from the class name.